openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.component.tsx

```diff
@@ -214,14 +215,20 @@ const Explore: React.FC<ExploreProps> = ({
         <Card
           className="page-layout-v1-left-panel page-layout-v1-vertical-scroll"
           data-testid="data-summary-container">
-          <FacetFilter
-            aggregations={omit(searchResults?.aggregations, 'entityType')}
-            filters={postFilter}
-            showDeleted={showDeleted}
-            onChangeShowDeleted={onChangeShowDeleted}
-            onClearFilter={onChangePostFilter}
-            onSelectHandler={handleFacetFilterChange}
-          />
+          <ExploreSkeleton
+            loading={
+              isUndefined(searchResults?.aggregations) &&
+              isUndefined(postFilter)
```

Comment:
what will happen if both stay undefined?

Reply:
It will show the skeleton.

Reply:
the screen will be stuck right if no data come from API?

Reply:
@ShaileshParmar11 , As I said skeleton will be shown, and the screen will not stuck.
We are using the antd Skeleton Component.

Reply:
Even after API fails or receives no data from API, it will keep showing a skeleton loader so this should not be happening right? user will think something is loading and he will be under the impression that something will come up
That's why I [mention](https://github.com/open-metadata/OpenMetadata/pull/9372#pullrequestreview-1221781542) please test it out with no data! Let me know if you need more clarity on this or want to connect on this!

https://user-images.githubusercontent.com/71748675/208242150-6c709015-49ac-4c7f-b1fa-fe7cf15e8212.mov

Reply:
@ShaileshParmar11 , I mentioned the same, it is showing the skeleton.

Reply:
I already tested it out and then answered your question that it will show the skeleton.

Reply:
it should not be like that right?
@chirag-madlani Please share your input